src/redux/reducers/index.ts

```diff
@@ -15,6 +15,21 @@ function rootReducer(state = initialState, action : Action) {
                 user : { name : sAction.payload.username}
             });
         }
+        case actions.ADD_WORKOUT : {
+            var addWorkout = action as AddWorkoutAction;
+            return {
+                ...initialState,
+                workouts: [addWorkout.payload.workoutName, ...initialState.workouts]
+            }
+        }
+        case actions.DELETE_WORKOUT : {
+            var deleteWorkout = action as deleteWorkoutAction
+            return {
+                ...initialState, 
+                workouts: initialState.workouts.filter(workout => deleteWorkout.payload.workoutName !== workout)
```

Comment:
We'll eventually also need to add the api request to update the workouts serverside